gravitee-apim-console-webui/src/management/configuration/configuration.route.ts

```diff
@@ -650,10 +649,6 @@ function configurationRouterConfig($stateProvider: StateProvider) {
     .state('management.settings.metadata', {
       url: '/metadata',
       component: 'metadata',
-      resolve: {
-        metadata: ['MetadataService', (MetadataService: MetadataService) => MetadataService.list().then((response) => response.data)],
-        metadataFormats: ['MetadataService', (MetadataService: MetadataService) => MetadataService.listFormats()],
-      },
       data: {
```

Comment:
useAngular are missing here. remember, this allow to deactivate the legacy css style

Reply:
Done